problems.txt

```diff
@@ -0,0 +1,96 @@
+2017-07-06T17:31:59.001Z pid:20915 worker:1 [2017-07-06 10:31:59.000] [INFO] knj_log - KNJ_AAR_BATCH_FREQ undefined
+2017-07-06T17:31:59.001Z pid:20915 worker:1 [2017-07-06 10:31:59.000] [INFO] knj_log - KNJ_AAR_BATCH_COUNT 100
+2017-07-06T17:31:59.001Z pid:20915 worker:1 [2017-07-06 10:31:59.000] [INFO] knj_log - KNJ_AAR_BATCH_FREQ 60
+2017-07-06T17:31:59.001Z pid:20915 worker:1 [2017-07-06 10:31:59.000] [INFO] knj_log - ==========End of inital BAM parameters setting==========
+2017-07-06T17:31:59.166Z pid:20915 worker:1 [2017-07-06 10:31:59.165] [INFO] knj_log - Configuration updated: deepdive is enabled
+2017-07-06T17:31:59.176Z pid:20915 worker:1 [2017-07-06 10:31:59.168] [ERROR] knj_restclient_log - register_topology set cofniguration failed. { Error: ENOENT: no such file or directory, open './config.json'
+2017-07-06T17:31:59.176Z pid:20915 worker:1     at Object.fs.openSync (fs.js:651:18)
+2017-07-06T17:31:59.176Z pid:20915 worker:1     at Object.fs.readFileSync (fs.js:553:33)
+2017-07-06T17:31:59.177Z pid:20915 worker:1     at Object.module.exports.setConfiguration (/home/sam/w/sn/strong-supervisor/node_modules/ibmapm/lib/knj-restclient/index.js:52:29)
+2017-07-06T17:31:59.177Z pid:20915 worker:1     at JsonSender.init (/home/sam/w/sn/strong-supervisor/node_modules/ibmapm/lib/json-sender.js:193:28)
+2017-07-06T17:31:59.177Z pid:20915 worker:1     at MetricManager.start (/home/sam/w/sn/strong-supervisor/node_modules/ibmapm/lib/metric-manager.js:87:20)
+2017-07-06T17:31:59.177Z pid:20915 worker:1     at MonitoringPlugin.init (/home/sam/w/sn/strong-supervisor/node_modules/ibmapm/lib/plugin.js:35:19)
+2017-07-06T17:31:59.177Z pid:20915 worker:1     at Object.<anonymous> (/home/sam/w/sn/strong-supervisor/node_modules/ibmapm/index.js:192:12)
+2017-07-06T17:31:59.177Z pid:20915 worker:1     at Module._compile (module.js:569:30)
+2017-07-06T17:31:59.181Z pid:20915 worker:1     at Object.Module._extensions..js (module.js:580:10)
+2017-07-06T17:31:59.182Z pid:20915 worker:1     at Module.load (module.js:503:32)
+2017-07-06T17:31:59.182Z pid:20915 worker:1   errno: -2,
+2017-07-06T17:31:59.182Z pid:20915 worker:1   code: 'ENOENT',
+2017-07-06T17:31:59.182Z pid:20915 worker:1   syscall: 'open',
+2017-07-06T17:31:59.182Z pid:20915 worker:1   path: './config.json' }
+2017-07-06T17:31:59.182Z pid:20915 worker:1 [2017-07-06 10:31:59.177] [ERROR] knj_restclient_log -  VCAP_APPLICATION env variable not found, faild to connect to AvailabilityMonitoring service.
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.177] [INFO] knj_restclient_log - No any service is bound for this application.
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.179] [ERROR] knj_restclient_log - No ingress URL is set, please set "ingressURL" in your configuration json file
+2017-07-06T17:31:59.183Z pid:20915 worker:1 [2017-07-06 10:31:59.180] [INFO] knj_log - Failed to get /home/sam/w/sn/strong-supervisor/bin//package.json
```

Comment:
this is a problem. we need to discuss how you are finding the package root. I suggest using https://nodejs.org/api/all.html#process_process_mainmodule